app/views/admin/submissions/show.html.erb

```diff
@@ -23,14 +23,16 @@
           </div>
           <%= render 'admin/consignments/consignment_section', consignment: @submission.consigned_partner_submission, artist: @submission.artist&.name %>
           <%= render 'admin/offers/offers_section', offers: @offers, truncated: true %>
-          <% if @notes.size > 0 %>
```

Comment:
This indent looks wrong to me. I think the conditional was removed, so I'd expect for the following block to be outdented, no? #minor